### PR TITLE
Revisions to SoF

### PR DESCRIPTION
**Pull request by doofus-01**
These are some revisions to first scenario so that the player is a little more involved, instead of just the AIs.  The scenario length and difficulty are not changed.  There are some minor changes to the map associated with the scenario changes.

### CODE REVIEW

**Comment by Vultraz** [comment]
Please run wmlindent on your changes. Also, since you added the snow detritus, you can delete the #TODO.

**Comment by doofus-01** [comment]
I think the first scenario can be reviewed now.  The AI settings have comments in them, because I'm having some trouble telling if the settings do anything and I want to keep them flagged.  The most obvious is the limits on recruitment, which I may have done wrong, but I'm not sure any of it is working.

File: data/campaigns/Sceptre_of_Fire/scenarios/2t_In_the_Dwarven_City.cfg [hunk 1]
```diff
@@ -149,18 +155,17 @@
...
-        [/recall]
         [message]
             speaker=Rugnur
-            message= _ "Hey, Durstorn, have we started yet? King Haldric came by just now, wanted to talk to you..."
+            message= _ "Hey, Durstorn, have we started yet?"
         [/message]
         [message]
             speaker=Durstorn
-            message= _ "Speak properly to your elders, boy! Why are you late?"
+            message= _ "Speak properly to your elders, boy! What are you taking about?  Why are you late?"
```
Comment (aquileia) [inline data/campaigns/Sceptre_of_Fire/scenarios/2t_In_the_Dwarven_City.cfg]: "What are you taking about?  Why are you late?"
--> "What are you talking about? Why are you late?"
('talking' and only a single space)

Comment (doofus-01) [inline data/campaigns/Sceptre_of_Fire/scenarios/2t_In_the_Dwarven_City.cfg]: Thanks, fixed.  The single space after a sentence is going to be a constant issue.

**Comment by doofus-01** [comment]
Changes to scenario 3 are cosmetic only.

**Comment by ln-zookeeper** [comment]
Don't the added elf spawns in S1 (mainly the northern spawn) present quite a tomato surprise? If you've sent the caravans along the road with your other units fighting with Glildur, having an Elvish Champion spawn 1-3 turns away from the caravans seems like an instant game over, unless you manage to kill Glildur first.

**Comment by doofus-01** [comment]
Revised scenario 1 to remove the turn 10 event.  Player should be involved, at about the same difficulty.  It may be too easy to kill elf leader.

**Comment by doofus-01** [comment]
I misunderstood what was needed and this campaign may get replaced anyway, so I will close this request if I can.

**Pull request closed without merging**